spec/models/post_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject { Post.new(title: 'A' * 251, likes_count: -1, comments_count: -1) }
+
+  it 'should be invalid if title is blank' do
+    subject.title = ''
+    expect(subject).to_not be_valid
+  end
+
+  it 'should be invalid if title is longer than 250 characters' do
+    expect(subject).to_not be_valid
+  end
+
+  it 'should be invalid if likes_count is less than 0' do
+    expect(subject).to_not be_valid
+  end
+
+  it 'should be invalid if comments_count is less than 0' do
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] Good job with test coverage, but since it is a [requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md)for this project, "`Add unit specs for all of your models' methods and validations`", please ensure all custom methods in the models have a corresponding unit test, methods like: `recent_comments`, `increment_likes_count!`, `increment_author_posts_counter`. Please kindly fix that.